genesyscloud/telephony_providers_edges_site_outbound_route/data_source_genesyscloud_telephony_providers_edges_site_outbound_route.go

```diff
@@ -0,0 +1,41 @@
+package telephony_providers_edges_site_outbound_route
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	featureToggles "terraform-provider-genesyscloud/genesyscloud/util/feature_toggles"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceSiteOutboundRouteRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	if exists := featureToggles.OutboundRoutesToggleExists(); !exists {
```

Comment:
Hey Brian is this a brand new data source.  If it then I don't think we need a feature toggle here.  Just  curious as to why you put one here.

Reply:
I included the matching feature toggle to the existing resource.